test/integration/crud/explain.test.ts

```diff
@@ -1,40 +1,29 @@
 import { expect } from 'chai';
 
-import {
-  type Collection,
-  type CommandStartedEvent,
-  type Db,
-  type MongoClient,
-  MongoServerError
-} from '../../mongodb';
+import { type Collection, type Db, type MongoClient, MongoServerError } from '../../mongodb';
 
 describe('Explain', function () {
   let client: MongoClient;
   let db: Db;
   let collection: Collection;
-  let commandsStarted: CommandStartedEvent[];
 
   beforeEach(async function () {
     client = this.configuration.newClient({ monitorCommands: true });
     db = client.db('queryPlannerExplainResult');
     collection = db.collection('test');
-    commandsStarted = [];
 
     await collection.insertOne({ a: 1 });
-    client.on('commandStarted', event => commandsStarted.push(event));
   });
 
   afterEach(async function () {
     await collection.drop();
     await client.close();
-    commandsStarted = [];
   });
 
   context('when explain is set to true', () => {
     it('deleteOne returns queryPlanner explain result', async function () {
       const explanation = await collection.deleteOne({ a: 1 }, { explain: true });
       expect(explanation).property('queryPlanner').to.exist;
-      expect(commandsStarted[0]).to.have.nested.property('command.explain.delete');
```

Comment:
We still want to assert on this, but since it's the only test that relies on the command started listener and events, we want to move that setup into this test, not remove it entirely.